mteb/tasks/Retrieval/eng/LEMBNeedleRetrieval.py

```diff
@@ -0,0 +1,104 @@
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class LEMBNeedleRetrieval(AbsTaskRetrieval):
+    _EVAL_SPLIT = [
+        "test_256",
+        "test_512",
+        "test_1024",
+        "test_2048",
+        "test_4096",
+        "test_8192",
+        "test_16384",
+        "test_32768",
+    ]
+
+    metadata = TaskMetadata(
+        name="LEMBNeedleRetrieval",
+        dataset={
+            "path": "dwzhu/LongEmbed",
+            "revision": "6e346642246bfb4928c560ee08640dc84d074e8c",
+            "name": "needle",
+        },
+        reference="https://huggingface.co/datasets/dwzhu/LongEmbed",
+        description=("needle subset of dwzhu/LongEmbed dataset."),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=_EVAL_SPLIT,
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=("2000-01-01", "2023-12-31"),
+        form=["written"],
+        domains=["Academic", "Blog"],
+        task_subtypes=["Article retrieval"],
+        license="Not specified",
+        socioeconomic_status="high",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation=None,
```

Comment:
Missing?

Reply:
these are two synthetic tasks, I guess it's not necessary to provide citations, as different people can synthetic different data?

Reply:
if must cite, I think I should cite my paper, which is on hold on arxiv not ...

Reply:
I would cite the paper that introduces the synthetic task - sounds like that it your paper (in which case I would add that).

Reply:
thanks for this! Since its publication has been delayed (its status is `on hold` in arxiv, which I have no idea will take how long), can I add these fields later?

Reply:
Sure let us just get this merged in

Reply:
Thanks so much to you guys!